sysid/src/main/native/include/sysid/view/DataSelector.h

```diff
@@ -12,17 +12,17 @@
 #include <vector>
 
 #include <glass/View.h>
-#include <glass/support/DataLogReaderThread.h>
+#include <wpi/DataLogReaderThread.h>
 #include <wpi/StringMap.h>
 
 #include "sysid/analysis/Storage.h"
 
 namespace glass {
-class DataLogReaderEntry;
 class Storage;
 }  // namespace glass
 
 namespace wpi {
```

Comment:
The other DataLog functions are in the wpi::log namespace, so let's be consistent with that.